requirements.txt

```diff
@@ -1,5 +1,5 @@
 farm==0.8.0
---find-links=https://download.pytorch.org/whl/torch_stable.html
+# --find-links=https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Temporary till we fix this https://github.com/deepset-ai/haystack/pull/1458/files#r709951364